src/Windowing/Silk.NET.SDL/SdlProvider.cs

```diff
@@ -0,0 +1,68 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+
+namespace Silk.NET.SDL
+{
+    /// <summary>
+    /// Singleton providing easy GLFW implementation access.
+    /// </summary>
+    public static class SdlProvider
+    {
+        /// <summary>
+        /// The flags with which to initialize SDL.
+        /// </summary>
+        public static uint InitFlags { get; set; } = //Sdl.InitTimer |
+            //Sdl.InitAudio |
+            Sdl.InitVideo |
+            Sdl.InitJoystick |
+            Sdl.InitHaptic |
+            Sdl.InitGamecontroller |
+            Sdl.InitEvents;
+        //Sdl.InitSensor |
+        //Sdl.InitNoparachute;
```

Comment:
Why the comments?

Reply:
To demonstrate what systems we don't initialize by default.